clients/go/pkg/justification/processor.go

```diff
@@ -51,12 +54,15 @@ func NewProcessor(v Validator) *Processor {
 
 // Process populates the given audit log request with the justification info from the given token.
 // If the token is empty, this function does nothing.
-func (p *Processor) Process(jvsToken string, logReq *api.AuditLogRequest) error {
-	if jvsToken == "" {
+func (p *Processor) Process(ctx context.Context, logReq *api.AuditLogRequest) error {
+	jvsToken, ok := logReq.Context.GetFields()[TokenHeaderKey]
+	jvsTokenStr := jvsToken.GetStringValue()
+	if !ok || jvsTokenStr == "" {
+		log.Println("don't find justification token in field 'context' of AuditLogRequest")
```

Comment:
No, please use the logger from context if needed. See example: https://github.com/abcxyz/lumberjack/blob/d33b0444e158f6add08e7cd71704aa6cf5719ef3/clients/go/pkg/audit/client.go#L139

Add a TODO with https://github.com/abcxyz/lumberjack/issues/257. We will want to enforce the token later on.

Reply:
Do you mean the jvs token won't be optional in the future? We will always be able to find the jvs token in the context of grpc requests?

Reply:
changed to zlogger